src/vs/workbench/contrib/terminal/browser/terminalProfileService.ts

```diff
@@ -34,7 +34,7 @@ export class TerminalProfileService implements ITerminalProfileService {
 	private _profilesReadyBarrier: AutoOpenBarrier;
 	private _availableProfiles: ITerminalProfile[] | undefined;
 	private _contributedProfiles: IExtensionTerminalProfile[] = [];
-	private _defaultProfileName?: string;
+	private _defaultProfileName: string | null = null;
```

Comment:
Could keep this and do `getValue<string | null>(...) ?? undefined` instead, to handle the conversion at a single point.